yaml2sbml/YamlModel.py

```diff
@@ -184,10 +184,13 @@ def is_set_time(self):
 
     def set_time(self,
                  time_variable: str):
-        self._yaml_model['time'] = [time_variable]
+        self._add_entry({'variable': time_variable}, 'time')
```

Comment:
Could this solution allow for more than one time variable?

Reply:
Good catch! It could. And it was wrong anyhow. I changed that!